tcp/spring-cloud-starter-stream-processor-tcp-client/README.adoc

```diff
@@ -1,5 +1,5 @@
 //tag::ref-doc[]
-= TCP Client
+= TCP Client as a processor which connects to a TCP server, sends/waits for data
```

Comment:
Does it really "wait" for data. We saw that the server can send data to the app at a rate that is uncorrelated to what the app sends to the server (and that is a good thing)